lib/localisation/src/preprocessing/keyence.rs

```diff
@@ -1 +1,48 @@
+use heapless::Vec;
 
+#[derive(PartialEq, Debug)]
+pub enum SensorChecks {
+    Acceptable,
+    Unnaceptable,
+}
+
+pub struct KeyenceAgrees {
```

Comment:
Quick Rust doc `///` comment here to explain what this does would be good

Reply:
So would we want it to store the previous sensor check and compare with that too? That then gets updated at the end of each run with the new one? To handle the 2 in a row.

Reply:
Yeah exactly, first time the two new keyence values disagree you'd set `self.last_agreement_status` to `false` and if they disagree a second time then you'd trigger the failure (and it would reset to `true` when they agree)

Reply:
It just wants you to add:

```rust
impl Default for KeyenceAgrees {
    fn default() -> Self {
        Self::new()
    }
}
```